packages/pwa-kit-react-sdk/scripts/utils.js

```diff
@@ -241,7 +240,7 @@ Utils.requestErrorMessage = {
         'You do not have permission to perform this actions.\nPlease double check your command to make sure the option values are correct.', //  wrong project name.
     code404:
         'Resource not found.\nPlease double check your command to make sure the option values are correct.', // wrong target name
-    code500: 'Internal Server Error. Please report this to Mobify support team.'
+    code500: 'Internal Server Error. Please report this to Salesforce support team.'
```

Comment:
is there a link for Salesforce support team? maybe we should add the link to the error

Reply:
It's just going to be help.salesforce.com